README.md

```diff
@@ -285,6 +287,33 @@ fastify.listen(3000, err => {
 })
 ```
 
+#### Example trusted tokens
+```js
+const fastify = require('fastify')()
+
+fastify.register(require('fastify-jwt'), { 
+  secret: 'foobar',
+  trusted: checkIfTokenIsTrusted
```

Comment:
```suggestion
  trusted: validateToken
```

Reply:
Personally, I'm inclined to prefer explicit/specific wording when naming variables and functions, but I do agree that a simpler tone is more inline with what is used in the rest of the documentation. I'm OK changing it.